app/Http/Requests/ReportRequest.php

```diff
@@ -32,15 +34,15 @@ public function rules()
             case 'POST':
             {
                 return [
-                    'name' => 'required|max:100|unique:reports,name',
+                    'name' => 'required|max:100|unique:reports,name,',
```

Comment:
thêm `,` làm gì thế

Reply:
syntax cuả nó để gọi sang thằng this đằng sau.